f2/src/PrivacyStatementPage.js

```diff
@@ -56,20 +100,21 @@ export const PrivacyStatementPage = () => {
               <Trans id="privacyStatementPage.toinquireText1" />
             </P>
             <P>
-              <Trans id="privacyStatementPage.toinquireText2" />
+              <Trans id="privacyStatementPage.toinquireText2">             
+              <A
+              href="mailto=atipb@rcmp-grc.gc.ca"
```

Comment:
these two `mailto` links don't seem to work for me.
I think it has to be `mailto:atipb@rcmp-grc.gc.ca`

Reply:
I think this is fixed now :)